homeassistant/components/qbittorrent/__init__.py

```diff
@@ -118,10 +117,13 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
             config_entry.data[CONF_PASSWORD],
             config_entry.data[CONF_VERIFY_SSL],
         )
-    except LoginRequired as err:
+    except LoginFailed as err:
         raise ConfigEntryNotReady("Invalid credentials") from err
-    except RequestException as err:
-        raise ConfigEntryNotReady("Failed to connect") from err
+    except Forbidden403Error as err:
+        raise ConfigEntryNotReady("Fail to log in, banned user ?") from err
```

Comment:
Shouldn't this cause us to start a reauth flow instead? But maybe that's for a follow-up?

Reply:
Maybe in another PR because I need to take a better look to the documentation, as I have seen, we need to update the Config flow to make this work right ?

Reply:
Yes, that's right, the config flow needs to be updated